### DISCUSSION TITLE
BR-BT-00137-0200 and the ESPD alfa button

### DISCUSSION THREAD

**Discussion by kletersteegp** [OP-TED/eForms-SDK#359] [Q&A] [answered]
BR-BT-00137-0200 states that you have to supply at least one selection criteria for the 3 types. 
Selection Criteria (BG-702) must be present three times and Selection Criteria Type (BT-747) value must be equal once to 'Suitability to pursue the professional activity', once to 'Economic and financial standing', and once to 'Technical and professional ability'

In Netherlands we are by law not allowed to ask this. For selection criteriria we use the alfa button as it is also used in the ESPD. Sincce i can't seem to upload an image hereby the text of step 4 in the ESPD. What would be the best way to implement the alfa button in the eForms Selection Criteria. So that is will not trigger bussines rule BR-BT-00137-0200.

Regards 
Peter van de Kletersteeg

_

> Part IV: Selection criteria
> Contracting authorities must indicate which selection criteria will be applied by 'ticking' the checkbox placed before the relevant criterion. Do you want to use the selection criteria from A to D? Yes No
> ɑ: Global indication for all selection criteria
> Concerning the selection criteria the contracting authority declares that 
> It satisfies all the required selection criteria indicated in the relevant notice or in the procurement documents referred to in the notice.

_

**Comment by YvesJo** [comment, marked as answer]
Hi Peter,
For this Business related concern, I invite you to get directly in touch with the helpdesk: ted@publications.europa.eu
You will also have the opportunity to attach the picture you wanted to share.
I mark this question as answered as it is not the right place for such discussion.
Thanks for your understanding,
Yves
